packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -180,27 +187,44 @@ export class FunctionService {
    */
   public async debugFunction(rootDir: string) {
     const functionExecution = this.getStartedFunction(rootDir);
-    if (functionExecution) {
-      const { debugPort, debugType } = functionExecution;
-      const debugConfiguration: vscode.DebugConfiguration = {
-        type: debugType,
-        request: 'attach',
-        name: 'Debug Invoke', // This name doesn't surface in UI
-        resolveSourceMapLocations: ['**', '!**/node_modules/**'],
-        console: 'integratedTerminal',
-        internalConsoleOptions: 'openOnSessionStart',
-        localRoot: rootDir,
-        remoteRoot: '/workspace',
-        hostName: '127.0.0.1',
-        port: debugPort
-      };
-      if (!functionExecution.debugSession) {
-        await vscode.debug.startDebugging(
-          getRootWorkspace(),
-          debugConfiguration
-        );
-      }
+    if (!functionExecution) {
+      // TODO: report error?
```

Comment:
Should we report an error here?  And the sting on line 62, should it be localized?

Reply:
Yes. Let's localize that message. I think this is something we missed.

Reply:
Except for this, everything looks good. I'll approve the PR now and you can merge after localizing this message and QA.

Reply:
done